modules/ROOT/pages/release-notes.adoc

```diff
@@ -8,10 +8,10 @@ The 2022.2 is in Beta.
 
 == New added values
 
-=== Bonita now embeds the bpmn-visualization library to improve process monitoring for administrators and users 
+=== Bonita process monitoring now with embeded bpmn-visualization library
 
-Bonitasoft has launched the open source project Process Analytics two years ago, as a response to market's need in bpmn process monitoring execution using bpmn standards. +
-The deliverable of this project is called the _bpmn-visualization library_ . You can find a nice array of examples of what it can do in its https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/v0.25.0/examples/index.html[demo web site], and you can even contribute to this initiative in its https://github.com/process-analytics/bpmn-visualization-js[Github repository]. +
+Bonitasoft launched the open source project https://process-analytics.dev/[Process Analytics] two years ago in response to the market need to perform BPMN process monitoring using BPMN standards.
+The deliverable of this project is called the _bpmn-visualization library_ . You can find a nice array of examples of what it can do in its https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/v0.25.2/examples/index.html[demo web site], and you can even contribute to this initiative in its https://github.com/process-analytics/bpmn-visualization-js[Github repository]. +
```

Comment:
```suggestion
The deliverable of this project is called the _bpmn-visualization library_. You can find a nice array of examples of what it can do in its https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/v0.25.2/examples/index.html[demo web site], and you can even contribute to this initiative in its https://github.com/process-analytics/bpmn-visualization-js[Github repository]. +
```